cli/tests/pcluster/configure/test_pcluster_configure.py

```diff
@@ -43,16 +43,46 @@ def _mock_aws_region(mocker):
     mocker.patch(EASYCONFIG + "get_regions", return_value=regions)
 
 
-def _mock_list_keys(mocker):
+def _mock_aws_china_region(mocker):
```

Comment:
instead of having two functions to mock the region we could define a dict and return the right list according to the partition value.

```python
def _mock_aws_region(mocker, partition="commercial"):
  regions = {
  "commercial" : [
        "eu-north-1",
        "ap-south-1",
         ...
    ] 
   "china" : [
        "cn-north-1",
        "cn-northwest-1",
         ...
    ] 
  }
  return regions.get(partition)
```

What do you think?

We can do the same for mock_list_keys, etc, instead of having a boolean flag.
It could be useful if in future we want to moke more partitions.